### PR TITLE
[GitHub] Update issue templates

### PR DESCRIPTION
**Pull request by concretevitamin**
To gather `sky -v / -c`.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/custom-issue-template.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+---
+name: Custom issue template
+about: Describe this issue template's purpose here.
+title: ''
+labels: ''
+assignees: ''
```
Comment (Michaelvll) [inline .github/ISSUE_TEMPLATE/custom-issue-template.md:6]: Should we fill in these fields? It seems when I click `New Issue`, these lines will show up first:
image

**Review by Michaelvll — COMMENTED** [review]
Thanks for adding this @concretevitamin! Left a comment : )

File: .github/ISSUE_TEMPLATE/custom-issue-template.md (hunk 1 above)
Comment (concretevitamin) [reply .github/ISSUE_TEMPLATE/custom-issue-template.md]: Good catch. Now made the template appear directly after clicking New Issue (just like our PR template):
Screen Shot 2024-03-31 at 18 31 18

**Review by Michaelvll — APPROVED** [review]
lgtm! Thanks!

**Pull request merged**